scripts/travis/setup-mistral.sh

```diff
@@ -6,8 +6,39 @@ if [ "$(whoami)" != 'root' ]; then
     exit 2
 fi
 
-MISTRAL_STABLE_BRANCH="st2-0.9.0"
+function version_ge() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)" == "$1"; }
+
+# Determine which mistral version to use
+if [[ "${TRAVIS_BRANCH}" == 'master' ]]; then
```

Comment:
I think it's good that we made `mistral` builds optional. So they have just optional informational purpose.